pkg/performanceprofile/controller/performanceprofile_controller_hypershift.go

```diff
@@ -71,3 +102,230 @@ func (r *PerformanceProfileReconciler) hypershiftSetupWithManager(mgr ctrl.Manag
 
 	return ctrl.NewControllerManagedBy(mgr).For(&corev1.ConfigMap{}, builder.WithPredicates(p)).Complete(r)
 }
+
+func (r *PerformanceProfileReconciler) hypershiftReconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+
+	if !ntconfig.InHyperShift() {
+		return reconcile.Result{}, fmt.Errorf("Using hypershift controller configuration while not in hypershift deployment")
+	}
+
+	instance := &corev1.ConfigMap{}
+
+	//REVIEW - Should be OperatorNamespace?
+	err := r.Get(ctx, req.NamespacedName, instance)
+	if err != nil {
+		if k8serrors.IsNotFound(err) {
+			//FIXME - In hypershift there is no "owned" objects. So how do we delete created objects? or do we even have to?
+			//jlom - as we create a configmap for each of the elements created from a PerformanceProfile and we reference
+			//       these configmaps in NodePool so they could be "transfered" to the managed cluster, maybe "delete" them is
+			//       as simple as derreference them in NodePool and delete the ConfigMaps.
+			//NOTE - Adding a label with PerformanceProfile name on it to the configmaps could make this look up easier.
```

Comment:
still a fixme? or figured out the outcome? let's fix the comment in a followup

Reply:
Figured out. 
OwnerRef is enough with all the created object but the RuntimeClass which needs to be explicitly deleted from hosted cluster (done in `hypershiftDeleteComponents`